src/landscape.cpp

```diff
@@ -1535,7 +1535,7 @@ bool GenerateLandscape(uint8_t mode)
 {
 	/* Number of steps of landscape generation */
 	static constexpr uint GLS_HEIGHTMAP = 3; ///< Loading a heightmap
-	static constexpr uint GLS_TERRAGENESIS = 5; ///< Terragenesis generator
+	static constexpr uint GLS_TERRAGENESIS = 4; ///< Terragenesis generator
```

Comment:
Why is this being changed? Related to the removed `IncreaseGeneratingWorldProgress(GWP_LANDSCAPE);` call in `tgp.cpp`?

Reply:
Yes. This is removed to prevent a redraw when the map is not yet in a valid state.